doc/FrequentTasks.md

```diff
@@ -0,0 +1,58 @@
+This is intended to document frequently executed tasks and provide
```

Comment:
Thanks for putting this together.  This is stuff I would have known a year ago and would have had to start from scratch with if I needed to do them tomorrow 👍